src/DssSpell.t.sol

```diff
@@ -336,6 +200,7 @@ contract DssSpellTest is DssSpellTestBase {
         assertTrue(totalGas <= 10 * MILLION);
     }
 
+    // @dev The specific date doesn't matter that much since function is checking for difference between warps
```

Comment:
nit: natspec format not used in tests and incorrectly formatted (`//` instead of `///`).
```suggestion
    // The specific date doesn't matter that much since function is checking for difference between warps
```